election-server/src/bin/http.rs

```diff
@@ -177,14 +178,26 @@ struct AppState {
     initial_weights: HashMap<AccountAddress, Amount>,
 }
 
+impl AppState {
+    /// Gets the weight assigned to an account.
+    fn get_account_weight(&self, account: &AccountAddress) -> u64 {
```

Comment:
Could we name this `get_account_initial_weight`.

I got confused thinking this is the "current" weight, taking into account delegations.